test/e2e/receive_test.go

```diff
@@ -1215,74 +1215,3 @@ func TestReceiveCpnp(t *testing.T) {
 	}, v)
 
 }
-
-func TestNewTenant(t *testing.T) {
```

Comment:
remove this test introduced in https://github.com/databricks/thanos/pull/100 which didn't capture the bug anyway